diff --git a/src/node/handler/SocketIORouter.js b/src/node/handler/SocketIORouter.js
index 0a7361f..baef25d 100644
--- a/src/node/handler/SocketIORouter.js
+++ b/src/node/handler/SocketIORouter.js
@@ -61,8 +61,9 @@ exports.setSocketIO = function(_socket) {
     // Broken: See http://stackoverflow.com/questions/4647348/send-message-to-specific-client-with-socket-io-and-node-js
     // Fixed by having a persistant object, ideally this would actually be in the database layer
     // TODO move to database layer
-    if(settings.trustProxy && client.handshake.headers['x-forwarded-for'] !== undefined){
-      remoteAddress[client.id] = client.handshake.headers['x-forwarded-for'];
+
+    if(settings.log_x_client_ip && client.handshake.headers['x-client-ip'] !== undefined){
+      remoteAddress[client.id] = client.handshake.headers['x-client-ip'].split(',')[0];
     }
     else{
       remoteAddress[client.id] = client.handshake.address;
diff --git a/src/node/utils/Settings.js b/src/node/utils/Settings.js
index 2c2f90b..a6cbb6d 100644
--- a/src/node/utils/Settings.js
+++ b/src/node/utils/Settings.js
@@ -192,6 +192,11 @@ exports.sessionKey = false;
 */
 exports.trustProxy = false;
 
+/*
+* Whether to log X-Client-IP header
+*/
+exports.log_x_client_ip = false;
+
 /* This setting is used if you need authentication and/or
  * authorization. Note: /admin always requires authentication, and
  * either authorization by a module, or a user with is_admin set */
