Index: etherpad-lite/src/node/handler/SocketIORouter.js
===================================================================
--- etherpad-lite.orig/src/node/handler/SocketIORouter.js
+++ etherpad-lite/src/node/handler/SocketIORouter.js
@@ -61,6 +61,8 @@ exports.setSocketIO = function(_socket)
     // TODO move to database layer
     if (settings.trustProxy && client.handshake.headers['x-forwarded-for'] !== undefined) {
       remoteAddress[client.id] = client.handshake.headers['x-forwarded-for'];
+    } else if(settings.log_x_client_ip && client.handshake.headers['x-client-ip'] !== undefined){
+      remoteAddress[client.id] = client.handshake.headers['x-client-ip'].split(',')[0];
     } else {
       remoteAddress[client.id] = client.handshake.address;
     }
Index: etherpad-lite/src/node/utils/Settings.js
===================================================================
--- etherpad-lite.orig/src/node/utils/Settings.js
+++ etherpad-lite/src/node/utils/Settings.js
@@ -246,6 +246,11 @@ exports.sessionKey = false;
 exports.trustProxy = false;
 
 /*
+ * Whether to log X-Client-IP header
+ */
+exports.log_x_client_ip = false;
+
+/*
  * This setting is used if you need authentication and/or
  * authorization. Note: /admin always requires authentication, and
  * either authorization by a module, or a user with is_admin set
