--- a/settings.json.template
+++ b/settings.json.template
@@ -171,7 +171,7 @@
 
   "dbType": "dirty",
   "dbSettings": {
-    "filename": "var/dirty.db"
+    "filename": "/var/lib/etherpad-lite/dirty.db"
   },
 
   /*
@@ -514,10 +514,9 @@
    */
   "logconfig" :
     { "appenders": [
-        { "type": "console"
-        //, "category": "access"// only logs pad access
+        { "type": "file"
+      , "filename": "/var/log/etherpad-lite/etherpad-lite.log"
         }
-
       /*
       , { "type": "file"
       , "filename": "your-log-file-here.log"
