Index: etherpad-lite/settings.json.template
===================================================================
--- etherpad-lite.orig/settings.json.template
+++ etherpad-lite/settings.json.template
@@ -69,7 +69,7 @@
    * credentials.json.
    */
   "dbSettings" : {
-                   "filename" : "var/dirty.db"
+                   "filename" : "/var/lib/etherpad-lite/dirty.db"
                  },
 
   /*
@@ -372,10 +372,9 @@
    */
   "logconfig" :
     { "appenders": [
-        { "type": "console"
-        //, "category": "access"// only logs pad access
+        { "type": "file"
+      , "filename": "/var/log/etherpad-lite/etherpad-lite.log"
         }
-
       /*
       , { "type": "file"
       , "filename": "your-log-file-here.log"
