--- a/settings.json.template	2014-08-01 16:07:26.182099939 +0000
+++ b/settings.json.template	2014-08-01 16:10:14.302084632 +0000
@@ -38,7 +38,7 @@
   "dbType" : "dirty",
   //the database specific settings
   "dbSettings" : {
-                   "filename" : "var/dirty.db"
+                   "filename" : "/var/lib/etherpad-lite/dirty.db"
                  },

   /* An Example of MySQL Configuration
@@ -130,16 +130,9 @@
   // You can add as many appenders as you want here:
   "logconfig" :
     { "appenders": [
-        { "type": "console"
-        //, "category": "access"// only logs pad access
-        }
-    /*
-      , { "type": "file"
-      , "filename": "your-log-file-here.log"
-      , "maxLogSize": 1024
-      , "backups": 3 // how many log files there're gonna be at max
-      //, "category": "test" // only log a specific category
-        }*/
+       { "type": "file"
+      , "filename": "/var/log/etherpad-lite/etherpad-lite.log"
+	}
     /*
       , { "type": "logLevelFilter"
         , "level": "warn" // filters out all log messages that have a lower level than "error"
